src/foam/build/DirCrawlModelDAO.js

```diff
@@ -84,7 +84,7 @@ foam.CLASS({
 
       context.foam.LIB = function(m) {
         if ( ! m.id ) {
-          m.id = 'lib.Lib' + Date.now()
+          m.id = 'lib.Lib' + foam.next$UID()
```

Comment:
Can we name libs of the first method/axiom or something?  Or could we use a local incrementing id in order to make the generated ids a bit more stable?

Reply:
Yup how does the name + first method name sound? I could add all method names to be ultra safe but that'll cause really long IDs.